src/pages/Bridge/BurnForm/index.tsx

```diff
@@ -163,7 +163,7 @@ const BurnForm = (): JSX.Element | null => {
           [WRAPPED_TOKEN_AMOUNT]: ''
         });
         setSubmitStatus(STATUSES.RESOLVED);
-      } catch (error) {
+      } catch (error: any) {
```

Comment:
@tomjeatt 
One question, what TypeScript version are you using? I guess it's not matched with the installed (package.json) version.

Reply:
VSCode ships with 4.5.4 - hadn't realised it doesn't use the workspace version by default. Have updated the workspace config to point to the version in node_modules

Reply:
@tomjeatt 
In that case, I think this is not complained by the compiler.

Reply:
Yes - good sleuthing :) I would recommend leaving these in though, as we'll want to upgrade TS at some point and this will prevent us triggering validation errors when we do.

Reply:
Yes, we may want to upgrade TS later on but I'd say we do not have to do like `error: any` now because:
- We do not know when we upgrade TS. It may not be necessarily required.
- Using `any` is an anti-pattern in TypeScript.

I'd suggest handling this together with TS upgrade. Not now.

Reply:
Yes sure - don't feel very strongly about it so happy to revert. Worth pointing out though that although using any is an anti-pattern in most cases, it isn't in this case as the only valid type annotation in the catch clause is `any` or `unknown`. 

The real issue here is that we're using `catch(error)` without a type annotation which means that the catch block is not being type checked and that's why it's being flagged as an error in more recent versions of typescript.

I'll add a ticket to the backlog, as we should ideally be running the latest version of TS in our projects.